sim/core/attack.go

```diff
@@ -127,10 +127,10 @@ func (weapon *Weapon) CalculateNormalizedWeaponDamage(sim *Simulation, attackPow
 }
 
 func (unit *Unit) MHWeaponDamage(sim *Simulation, attackPower float64) float64 {
-	return unit.AutoAttacks.mh.CalculateWeaponDamage(sim, attackPower)
+	return unit.AutoAttacks.mh.CalculateWeaponDamage(sim, attackPower) * (1 + unit.PseudoStats.MHDpsMultiplier)
```

Comment:
Now that I think about it, rather than creating a multiplier pseudostat for this, couldn't you just multiply `unit.AutoAttacks.mh.BaseDamageMin` and `unit.AutoAttacks.mh.BaseDamageMax` dynamically?

Reply:
Potentially creating a method `(weapon *Weapon) ModWeaponDamage(multiplier float64)` to do this?